### DISCUSSION TITLE
Ability to search for cable labels

### DISCUSSION THREAD

**Discussion by themmini** [netbox-community/netbox#8059] [Ideas]
Ability to search for labels on ports for patch panels - patch ports are labeled in a datacentre with a serial number. Ability to search DCIM for that label when performing operations + audits/reconciliation.

Not sure if someone has an alternative method.

**Comment by candlerb** [comment]
Cables, interfaces and frontports can all have "labels" set on them.  Global search doesn't work across these, but it works if you search directly on them (e.g. go to Devices > Front Ports, Filter)

Are you therefore saying that you want these to be available in global search?  It seems reasonable to me, but similar requests have been rejected before: see #6798, #4757.  Open issues which may be relevant: #7016, #6003.

As a workaround, maybe you could write a Custom Script which prompts you for a keyword, and then searches labels across cables, interfaces, frontports and rearports.

**Comment by themmini** [comment]
Global search would be handy - for example labeling all the ports across your organisation then searching for that label to find the two ports that are connected. I guess you can search across all patch panels.
________________________________

**Comment by empusas** [comment]
I find it confusing to even show the cable ID. The ID is a consecutive number assigned when the object is created that has very limited practical use for the user. If you delete and recreate the same cable, it gets a new ID assigned and you got zero influence on the ID.
Using the label instead would be a much better option as this is usually assigned by the cabling team that physically marks the cable exactly with that.

**Reply by candlerb** [reply]
That's exactly what happens today. The cable ID is only shown for cables with no label.  If you set the label, that is shown instead.
